eve/main.yml

```diff
@@ -360,6 +360,17 @@ stages:
             /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s/bootstrap.sh
             --verbose
           haltOnFailure: true
+      - ShellCommand: *install_kubectl
```

Comment:
It's not needed. Install of kubectl is part of the bootstrap "now". Btw this should be removed from every step